### PR TITLE
Add plugin

### PR DESCRIPTION
**Pull request by mmcfarland**
#### Overview
Basic demo that attempts to animate flood layers on a 3D enabled SceneView.

Connects #2 

Features:
* Calls into new framework methods to turn on/off 2d and 3d views
* Flys to locations
* Adds map layers over 3d terrain 

The "animation" effect is underwhelming, mostly due to the time that it takes to "drape" the image over the terrain.  On the whole, the experience is pretty choppy.  This is intended as less an important tool rather than an exmple of how to use these new features in the context of the framework.

In conjunction with https://github.com/CoastalResilienceNetwork/GeositeFramework/pull/869

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+                $display.empty();
+                removeLayer(self.currentLayer);
+                var animationStep = this.zoomTo(location);
+                layerIds.forEach(function(id) {
+                    animationStep = animationStep
+                        .then(showLayer(id))
+                        .then(pause(pauseTime));
+                });
+                return animationStep;
```
Comment (kdeloach) [inline main.js:180]: Consider disabling the "Show Flood Layers" button while the animation is in progress, and maybe adding some kind of "loading" indicator? The layers don't load particularly fast, so having more visual cues could be helpful.

**Review by kdeloach — APPROVED** [review]
+1 Pretty cool.

File: main.js (hunk 1 above)
Comment (mmcfarland) [reply main.js:180]: Yeah, my initial intention was to wire up a slider that has a tick for each step in the animation so it moves forward and you see progress, however, due to how choppy it ended up being, I wasn't sure it was worth the extra effort.  I'll disable the button though to give a little better cue.

**Comment by mmcfarland** [comment]
Since the `animateLayers` function already returned a promise, I used that to add/remove the disabled prop on the button.

**Pull request merged**